pom.xml

```diff
@@ -506,6 +502,13 @@
                       <bannedImport>org.apache.commons.lang.**</bannedImport>
                     </bannedImports>
                   </restrictImports>
+                  <restrictImports implementation="de.skuzzle.enforcer.restrictimports.rule.RestrictImports">
+                    <includeTestCode>true</includeTestCode>
+                    <reason>Use edu.umd.cs.findbugs.annotations</reason>
```

Comment:
Sorry but this one got missed on initial commit @stoty.  Why did we go with the umd one instead of like the javax.validation.constraints one?

Reply:
NP, such stuff happens, @dbwong .

AFAIU javax.validation.constraints is for runtime validation, when the constraint is actively checked during runtime. (used mostly by IoC / DI stuff like Spring / Java EE)

Using that here would be confusing, since we only use these annotations to indicate intent, and to help with static code analysis (and I don't think findbugs would even understand that).

Reply:
> used mostly by IoC / DI stuff like Spring / Java EE

Yes makes sense